packages/forms/test/form_group_spec.ts

```diff
@@ -981,6 +1014,531 @@ describe('FormGroup', () => {
+           {errors: null, pending: false, status: 'VALID'},  // Control
+         ]);
+       }));
+
+    it('should handle multiple FormGroup validators (success then failure) resolving after successful child validator',
+       fakeAsync(() => {
+         const c = new FormControl(
+             'fcValue', null!, simpleAsyncValidator({timeout: 1, shouldFail: false}));
+         const g = new FormGroup({'one': c}, null!, [
+           simpleAsyncValidator({timeout: 2, shouldFail: false}),
+           simpleAsyncValidator({timeout: 3, shouldFail: true})
+         ]);
+
+         // Initially, the form group and nested control are in pending state
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: null, pending: true, status: 'PENDING'},  // Group
+           {errors: null, pending: true, status: 'PENDING'},  // Control
+         ]);
+
+         tick(1);
+
+         // After 1ms, only form control validation has resolved
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: null, pending: true, status: 'PENDING'},  // Group
+           {errors: null, pending: false, status: 'VALID'},   // Control
+         ]);
+
+         tick(1);
+         // After 1ms, one form async validator has resolved but not the second
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: null, pending: true, status: 'PENDING'},  // Group
+           {errors: null, pending: false, status: 'VALID'},   // Control
+         ]);
+
+         tick(1);
+
+         // After 1ms, the form group validation fails
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: {async: true}, pending: false, status: 'INVALID'},  // Group
+           {errors: null, pending: false, status: 'VALID'},             // Control
+         ]);
+       }));
+
+
+    it('should handle multiple FormGroup validators (failure then success) resolving after successful child validator',
+       fakeAsync(() => {
+         const c = new FormControl(
+             'fcValue', null!, simpleAsyncValidator({timeout: 1, shouldFail: false}));
+         const g = new FormGroup({'one': c}, null!, [
+           simpleAsyncValidator({timeout: 2, shouldFail: true}),
+           simpleAsyncValidator({timeout: 3, shouldFail: false})
+         ]);
+
+         // Initially, the form group and nested control are in pending state
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: null, pending: true, status: 'PENDING'},  // Group
+           {errors: null, pending: true, status: 'PENDING'},  // Control
+         ]);
+
+         tick(1);
+
+         // After 1ms, only form control validation has resolved
+         expect(currentStateOf([g, g.get('one')!])).toEqual([
+           {errors: null, pending: true, status: 'PENDING'},  // Group
+           {errors: null, pending: false, status: 'VALID'},   // Control
+         ]);
+
+
+         tick(1);
```

Comment:
```suggestion

         tick(1);
```
(removed extra line)